Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -691,6 +566,268 @@ inline void SeedFinder<external_spacepoint_t, platform_t>::filterCandidates(
   }  // loop on bottoms
 }
 
+template <typename external_spacepoint_t, typename platform_t>
+inline void SeedFinder<external_spacepoint_t, platform_t>::
+    filterCandidatesDetailedMeasurements(
```

Comment:
how much of this is a copy-paste of the `filterCandidates` method?

Reply:
Thank you for the comments! I think there is some duplicated code that can be avoided, I will work on that